spring-integration-jdbc/src/test/java/org/springframework/integration/jdbc/channel/PostgresChannelMessageTableSubscriberTests.java

```diff
@@ -300,4 +322,19 @@ public JdbcChannelMessageStore jdbcChannelMessageStore(DataSource dataSource) {
 
 	}
 
+	private static class ConnectionSupplier implements PgConnectionSupplier {
+		Runnable onGetConnection;
```

Comment:
Every member in the class must be surrounded with blank lines.
Including the last method in the class. 

Run `gradlew :spring-integration-jdbc:check`  locally.

Reply:
fixed